beacon-chain/cache/active_indices.go

```diff
@@ -16,7 +16,7 @@ var (
 	ErrNotActiveIndicesInfo = errors.New("object is not a active indices list")
 
 	// maxActiveIndicesListSize defines the max number of active indices can cache.
-	maxActiveIndicesListSize = 4
+	maxActiveIndicesListSize = 100
 
```

Comment:
why `400`?  this is going to use a lot of memory, and we likely don't care active indices that's older than previous epoch